src/components/message/__internal__/type-icon/type-icon.style.ts

```diff
@@ -12,9 +12,7 @@ const messageVariants = {
 };
 
 type TypeIconStyleProps = {
-  /** set background to be invisible */
   transparent?: boolean;
-  /** set type of message based on new DLS standard */
```

Comment:
**comment**: idk if this is particularly needed as part of the refactor, Its definitely cleaner but it may be better to only make test changes/code changes to enable said tests e.g adding `data-role` locators